### PR TITLE
Switched docs auto-deployment: pipenv => poetry

### PR DESCRIPTION
**Pull request by hrshdhgd**
Follows #720.

### CODE REVIEW

**Review by sujaypatil96 — APPROVED** [review]
All changes look good @hrshdhgd. Just need to wait for PR #720 to be reviewed and merged before merging this one in?

**Review by cmungall — APPROVED** [review]

**Pull request merged**
